instance/tasks.py

```diff
@@ -66,6 +66,7 @@ def watch_pr():
             instance, created = OpenEdXInstance.objects.get_or_create(
                 sub_domain=pr_sub_domain,
                 fork_name=pr.fork_name,
+                github_pr_number=pr.number,
```

Comment:
@antoviaque Good point, but I'm not sure about adding the PR number in the migration code.  I somehow don't like calling out to the Github API in migration code.  This feels too fragile to me.

I resorted to the very simple solution of setting the PR number a few lines further down.  The keys that are currently in the `get_or_create()` call are enough to uniquely identify the instance, so we can just unconditionally overwrite the PR number on the instance.